webapp/src/main/webapp/src/locale/messages.pt.xlf

```diff
@@ -509,151 +509,151 @@
-				<target></target>
+				<target>Índice de pesquisa atualizado</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-verify-running">
 			<segment>
 				<source>Verify task is running</source>
-				<target></target>
+				<target>Verifique se a tarefa está em execução</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-resolution">
 			<segment>
 				<source>Conflict Resolution</source>
-				<target></target>
+				<target>Resolução de Conflitos</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-prop-name">
 			<segment>
 				<source>Property Name</source>
-				<target></target>
+				<target>Nome da propriedade</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-current-value">
 			<segment>
 				<source>Current Value</source>
-				<target></target>
+				<target>Valor actual</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-new-value">
 			<segment>
 				<source>New Value</source>
-				<target></target>
+				<target>Novo valor</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-sync-qn">
 			<segment>
 				<source>Sync?</source>
-				<target></target>
+				<target>Sincronizar?</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-decision-required">
 			<segment>
 				<source>Decision is required</source>
-				<target></target>
+				<target>É necessário decidir</target>
 			</segment>
 		</unit>
 		<unit id="common-submit">
 			<segment>
 				<source>Submit</source>
-				<target></target>
+				<target>Submeter</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-resolve">
 			<segment>
 				<source>Resolve</source>
-				<target></target>
+				<target>Resolver</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-merge">
 			<segment>
 				<source>Merge</source>
-				<target></target>
+				<target>Unificar</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-sync-new">
 			<segment>
 				<source>Sync New State</source>
-				<target></target>
+				<target>Sincronizar novo estado</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-ignore-new">
 			<segment>
 				<source>Ignore New State</source>
-				<target></target>
+				<target>Ignorar novo estado</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-resolve-success">
 			<segment>
 				<source>Resolved successfully</source>
-				<target></target>
+				<target>Resolvido com sucesso</target>
 			</segment>
 		</unit>
 		<unit id="receiver-conflict-resolve-fail">
 			<segment>
 				<source>An error occurred while resolving the conflict</source>
-				<target></target>
+				<target>Ocorreu um erro ao resolver o conflito</target>
 			</segment>
 		</unit>
 		<unit id="common-ok">
 			<segment>
 				<source>Ok</source>
-				<target></target>
+				<target>Ok</target>
 			</segment>
 		</unit>
 		<unit id="common-legend">
 			<segment>
 				<source>Legend</source>
-				<target></target>
+				<target>Legenda</target>
```

Comment:
By the way, `Legend` here means key or glossary, do you use the same word in Portuguese?